client/extensions/woocommerce/app/store-stats/utils.js

```diff
@@ -127,6 +155,8 @@ export function formatValue( value, format, code ) {
 			return formatCurrency( value, code );
 		case 'number':
 			return Math.round( value * 100 ) / 100;
+		case 'percent':
+			return translate( '%(percentage)s%% ', { args: { percentage: value }, context: 'percent' } );
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( '%(percentage)s%% completed' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.